### PR TITLE
Fix appveyor builds and failing tests

### PR DESCRIPTION
**Pull request by wieslawsoltes**

### CODE REVIEW

File: src/Perspex.SceneGraph/RelativeRect.cs [hunk 1]
```diff
@@ -191,14 +191,14 @@ public static RelativeRect Parse(string s, CultureInfo culture)
...
-                    scale = 0.01;
+                    scale = 0.01m;
                 }
 
                 return new RelativeRect(
-                    double.Parse(parts[0], culture) * scale,
-                    double.Parse(parts[1], culture) * scale,
-                    double.Parse(parts[2], culture) * scale,
-                    double.Parse(parts[3], culture) * scale,
+                    (double)(decimal.Parse(parts[0], culture) * scale),
```
Comment (grokys) [inline src/Perspex.SceneGraph/RelativeRect.cs]: This seems strange to me. Why parse as a decimal and convert to double?

Comment (grokys) [inline src/Perspex.SceneGraph/RelativeRect.cs]: It's so the test passes right? I'd suggest fixing the test to only check x significant digits, or use a value that has a strict FP representation rather than doing this.

Comment (wieslawsoltes) [inline src/Perspex.SceneGraph/RelativeRect.cs]: It's so the test passes. What precision you suggest I use for double value comparisons?

Comment (wieslawsoltes) [inline src/Perspex.SceneGraph/RelativeRect.cs]: Ok removed decimal. I will create custom RelativeRect comparer, something like this: https://github.com/Perspex/Perspex/blob/01562bd98c20fec2608fcdc70c5e0cbed5a81c8b/tests/Perspex.Direct2D1.UnitTests/RectComparer.cs
https://github.com/Perspex/Perspex/blob/01562bd98c20fec2608fcdc70c5e0cbed5a81c8b/tests/Perspex.Direct2D1.UnitTests/Controls/Shapes/PathTests.cs#L13

**Comment by wieslawsoltes** [comment]
I have fixed Parse method for RelativeRect to use double and fixed tests by using custom RelativeRect comparer.

File: src/Perspex.SceneGraph/Rect.cs [hunk 2]
```diff
@@ -297,7 +297,8 @@ public override bool Equals(object obj)
         {
             if (obj is Rect)
             {
-                return this == (Rect)obj;
+                var other = (Rect)obj;
+                return Position == other.Position && Size == other.Size;
```
Comment (grokys) [inline src/Perspex.SceneGraph/Rect.cs]: What is the reason for this change? `operator ==` just does `Position == other.Position && Size == other.Size` so it looks like it's just doing the same thing.

Comment (wieslawsoltes) [inline src/Perspex.SceneGraph/Rect.cs:301]: To be the same as for Point:
https://github.com/wieslawsoltes/Perspex/blob/ci/src/Perspex.SceneGraph/Point.cs#L202

Comment (grokys) [inline src/Perspex.SceneGraph/Rect.cs]: Ok, fair enough ;)
